src/cascadia/TerminalApp/PaneArgs.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include "BellEventArgs.g.h"
+
+namespace winrt::TerminalApp::implementation
+{
+    struct BellEventArgs : public BellEventArgsT<BellEventArgs>

```

Comment:
Uhg so I had this here in a separate file so that when there is other content we implement, they could `#include "PaneArgs.h"` to get at the implementation without including the whole of `TerminalPaneContent.h`

Reply:
I'll move it though cause I don't think I used this anywhere in the stack up to the markdown pane, so it's not important yet